dev/package-examples/system-0.9.0/dataset/cpu/agent/input/input.yml

```diff
@@ -0,0 +1,4 @@
+- type: metric/system
```

Comment:
@ph I keep coming back that this here should be an input and not a stream. Reason is that a stream combines input + output, but this one here is only input configuration. How do you call it on the code side in the agent?